x-pack/plugins/observability/public/services/get_news_feed.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { AppMountContext } from 'kibana/public';
+
+export interface NewsItem {
+  title: { en: string };
+  description: { en: string };
+  link_url: { en: string };
+  image_url?: { en: string } | null;
+}
+
+interface NewsFeed {
+  items: NewsItem[] | [];
+}
+
+export async function getNewsFeed({ core }: { core: AppMountContext['core'] }): Promise<NewsFeed> {
+  try {
+    return await core.http.get('https://feeds.elastic.co/observability-solution/v8.0.0.json');
+  } catch (e) {
+    return { items: [] };
```

Comment:
You should also log the error so it's possible to debug errors

Reply:
@sqren  is a `console.error` enough?

Reply:
Yes, I think that's fine. We don't have any proper client side logging - can you believe? :D